src/ports/catalog/queries.ts

```diff
@@ -506,14 +506,14 @@ const getMaxPriceCase = (filters: CatalogQueryFilters) => {
 const getMaxPriceCaseWithTrades = (filters: CatalogQueryFilters) => {
   return filters.onlyMinting
     ? SQL`CASE
-                WHEN items.available > 0 AND items.search_is_store_minter = true 
+                WHEN items.available > 0 AND (items.search_is_store_minter = true OR items.search_is_marketplace_v3_minter = true)
```

Comment:
This should be fine if we don't check for open item trades to exist, as if it doesn't it would take `open_item_trade_price` as 0.

Reply:
change rollbacked, we don't need to cover it in the `getMaxPriceCase`, it should only go in the `getMaxPriceCaseWithTrades`